blackduck/HubRestApi.py

```diff
@@ -151,6 +159,12 @@ def get_auth_token(self):
             except json.decoder.JSONDecodeError as e:
               logger.exception("Authentication failure, could not obtain bearer token")
               raise Exception("Failed to obtain bearer token, check for valid authentication token")
+            length = len(response.text)
+            new_token = response.text
+            token_tail = new_token[length - 50: length]
+            print ("New token requested, using restconfig api token: {}".format(token_tail))
```

Comment:
We probably shouldn't be printing out the user's api token, same thing for the password below

Reply:
Agreed, I'll go ahead and remove them. My wording is not clear. The actual text is from the bearer token which is available from the response. Not the private Black Duck api token from rest config.

Reply:
And if you are going to be printing the token, don't do so using print - make it a call to `logger.debug(...)` or `logger.info(...)`, so it may at least be disabled by tweaking the log level.